pkg/storage/snapshot/snapshot.go

```diff
@@ -650,7 +650,7 @@ func (ctrl *VMSnapshotController) getVolumeSnapshotClass(storageClassName string
 	}
 
 	if len(matches) == 0 {
-		log.Log.Warningf("No VolumeSnapshotClass for %s", storageClassName)
+		log.Log.V(3).Infof("No VolumeSnapshotClass for %s", storageClassName)
```

Comment:
This change sounds very reasonable in the context of one of the calling function which updates the vmsnapshotstatus on the vm and so this function is called a lot!
I think we should add this Warning to line 632 before returning nil,nil in function getSnapshotPVC to get this log for better debug (since getSnapshotPVC is called in the vmsnapshot process it will not flood the log)

Reply:
@ShellyKa13 - I can move this logging if required. Is this a request to do so? I wasn't sure if it was a request or a suggestion that someone else would be giving a thumbs up for.

Reply:
I think you should leave the logging here with level 3 and also add a warning to line 632 to have more context in the snapshot functionallity which there it is important to now the reason.

Reply:
That makes sense. I have reinstated the original log message at the location suggested.